vnext/Microsoft.ReactNative/Fabric/Composition/TextInput/WindowsTextInputComponentView.cpp

```diff
@@ -541,7 +541,13 @@ void WindowsTextInputComponentView::HandleCommand(
     std::optional<winrt::hstring> text;
 
     winrt::Microsoft::ReactNative::ReadArgs(args.CommandArgs(), eventCount, text, begin, end);
-    if (eventCount >= m_nativeEventCount) {
+    // Accept text updates that match current event count, or clear text operations
+    // that are one event behind (to handle immediate setValue('') during onSubmitEditing)
+    bool isEmptyTextUpdate = text.has_value() && text.value().empty();
+    bool isValidEventCount = eventCount >= m_nativeEventCount;
+    bool isRecentClearText = isEmptyTextUpdate && (eventCount >= m_nativeEventCount - 1);
```

Comment:
This is still breaking the synchronization logic.    What if they do two setValue calls from JS.  Or 3.  Then the event count would be 2 or 3 values off.